pom.xml

```diff
@@ -21,12 +21,12 @@
     <!-- Artifact identification -->
     <groupId>io.opentracing.contrib</groupId>
     <artifactId>opentracing-spanmanager</artifactId>
-    <version>0.0.6-SNAPSHOT</version>
+    <version>0.1.0-SNAPSHOT</version>
```

Comment:
Given the 'work in progress, this isn't stable yet' status of this module I'd feel more comfortable with the 0.0.x versions.